PackageInfo.g

```diff
@@ -175,6 +260,47 @@ Persons := [
     Place         := "Siegen",
     Institution   := "Universität Siegen"
   ),
+ rec(
+    LastName      := "Siccha",
+    FirstNames    := "Sergio",
+    IsAuthor      := false,
+    IsMaintainer  := false,
+    Email         := "sergio@mathb.rwth-aachen.de",
+    WWWHome       := "https://www.mathematik.rwth-aachen.de/go/id/bkbg/gguid/0x28CF75713F0B7744BEF1377FB3F6748E/ikz/11/allou/1/lidx/1/",
+    PostalAddress := Concatenation(
+                       "Sergio Siccha\n",
+                       "Lehr- und Forschungsgebiet Algebra",
+                       "Pontdriesch 10-16\n",
+                       "Aachen\n",
+                       "Germany" ),
+    Place         := "Aachen",
+    Institution   := "RWTH Aachen University"
+  ), 
+  rec(
+    LastName      := "Wilson",
+    FirstNames    := "Wilf",
+    IsAuthor      := false,
+    IsMaintainer  := false,
+    Email         := "wilfred.wilson@mathematik.uni-halle.de",
```

Comment:
Please can you use my email and webpage as in:

https://github.com/gap-packages/Digraphs/blob/master/PackageInfo.g

Seeing as my employment in Halle is ending imminently, this data will be out-of-date very soon, so I’d prefer the more enduring options as used in the Digraphs package.

The same is true for the PostalAddress/Place/Institution, but I don’t have anything better to give for those, since I don’t yet know where my next job will be. So just leave as they are, or perhaps leave those fields out for now?

Reply:
Do you want me to remove `Place` and `Institution` for now?

Reply:
Yeah.